src/sentry/api/endpoints/project_transaction_threshold.py

```diff
@@ -78,8 +78,8 @@ def post(self, request, project):
 
         data = serializer.validated_data
 
-        with transaction.atomic():
-            try:
```

Comment:
Linking a thread from a different PR pertaining to these changes:
https://github.com/getsentry/sentry/pull/26824/files#r657054017